goal_src/levels/sunken/helix-water.gc

```diff
@@ -5,3 +5,782 @@
 ;; name in dgo: helix-water
 ;; dgos: L1, SUN, SUNKEN
 
+;; DECOMP BEGINS
+
+(deftype helix-slide-door (process-drawable)
+  ((root-override collide-shape  :offset 112)
+   )
+  :heap-base #x40
+  :method-count-assert 20
+  :size-assert         #xb0
+  :flag-assert         #x14004000b0
+  (:states
+    helix-slide-door-close
+    helix-slide-door-idle-closed
+    helix-slide-door-idle-open
+    )
+  )
+
+;; Manually added!
+(define-perm *helix-slide-door* helix-slide-door #f)
```

Comment:
I think we should make these `(define-extern *helix-slide-door* helix-slide-door)`. The `define-perm` version will set the value to `#f`, which didn't happen in the original game.